include/beast/http/parser_v1.hpp

```diff
@@ -62,6 +93,7 @@ class parser_v1
     std::string value_;
     message_type m_;
     typename message_type::body_type::reader r_;
+    std::uint8_t skip_body_ = 0;
```

Comment:
Got it. And case (2) Isn't implemented right now? I only see `skip_body_` taking on values on 1 and 0.